wikimedia/wikipedia-ios
# Remove unneeded language component from POTD call, add test

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T270139

### Notes
* Removes an unneeded helper function.

### Test Steps
1. Open picture of the day from Explore feed or widget, ensure it loads as expected.
2. Run new test, ensure it passes.
